### PR TITLE
KFSPTS-8394 Reimplement DV foreign draft/wire transfer handling

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

**Comment by db483** [comment]
Hi Chad,

I committed a temporary change to the pom but the KDO was not resolved yet so unit tests are still failing. Please DO NOT merge this PR before the pom change is reverted. Thank you!

**Comment by db483** [comment]
retest this please

**Comment by cah292** [comment]
retest this please

**Comment by cah292** [comment]
retest this please

**Comment by cah292** [comment]
retest this please

**Comment by cah292** [comment]
retest this please

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <kfs.version>2017-05-11</kfs.version>
         <rice.version>2.6.0</rice.version>
         <cynergy.version>2.6.0</cynergy.version>
-        <cynergy.version.suffix>current</cynergy.version.suffix>
```
Comment (cah292) [inline pom.xml]: Please undo this temporary POM change; I think the unit tests have been updated so that this change is no longer needed.

**Comment by db483** [comment]
Thank you Chad. I reverted the change to the pom file.

**Pull request merged**